Document use of undocumented parameters/properties (#1483)
stripe/stripe-go: README.md

c.Customer.ID    // ID is still available
c.Customer.Name  // Name is now also available (if it had a value)
```
### How to use undocumented parameters and properties

stripe-go is a typed library and it supports all public properties or parameters.

> Stripe sometimes has beta which introduces new properties or parameters that are not immediately public. The library does not support these properties or parameters until they are public but there is still an approach that allows you to use them.

Reviewer: ```suggestion
Stripe sometimes launches private beta features which introduce new properties or parameters that are not immediately public. These will not have typed accessors in the stripe-go library but can still be used. 
```

Reviewer: This will be changing as we introduce support for betas.

Author: This makes sense and in light of those coming changes the revised language seems much more informative.

#### Parameters

> To pass undocumented parameters to Stripe using stripe-go you need to use the `putExtraParam()` method, as shown below:

Reviewer: `putExtraParam` here should be `AddExtra` I believe?

```go

	params := &stripe.CustomerParams{
		Email: stripe.String("jenny.rosen@example.com")
	}

	params.AddExtra("secret_feature_enabled", "true")
	params.AddExtra("secret_parameter[primary]","primary value")
	params.AddExtra("secret_parameter[secondary]","secondary value")

	customer, err := customer.Create(params)
```

#### Properties

> To retrieve undocumented properties from Stripe using Go you can use an option in the library to return the raw JSON object and return the property. An example of this is shown below:

Reviewer: ```suggestion
You can access undocumented properties returned by Stripe by querying the raw response JSON object. An example of this is shown below:
```
